R/calcAreaEquippedForIrrigation.R

```diff
@@ -0,0 +1,113 @@
+#' @title calcAreaEquippedForIrrigation
+#' @description Calculates the area equipped for irrigation based on LU2v2 or
+#'              Mehta data sets.
+#'              For LUH2v2, it assumes, that all cropland irrigated in the last
+#'              20 years at least once is equipped for irrigation.
+#'              Mehta et al. (2022) directly report Global Area Equipped for
+#'              Irrigation for the years 1900-2015
+#'
+#' @param cellular    if TRUE: 0.5 degree resolution returned
+#' @param cells       number of cells to be returned: magpiecell (59199), lpjcell (67420)
+#' @param selectyears default on "past"
+#'
+#' @return List of magpie objects with results on country/cellular level,
+#'         weight on country level, unit and description.
+#'
+#' @author Benjamin Leon Bodirsky, Kristine Karstens, Felicitas Beier
+#'
+#' @seealso
+#' [calcLanduseInitialisation()]
+#' @examples
+#' \dontrun{
+#' calcOutput("AreaEquippedForIrrigation", source = "LUH2v2", cellular = TRUE, aggregate = FALSE)
+#' }
+#' @importFrom luscale groupAggregate
```

Comment:
`groupAggregate` does not seem to be used. Please remove this import if possibe

Reply:
Done. Thanks for spotting this!